tests/ui/test_ui_tools.py

```diff
@@ -933,6 +934,7 @@ def test_keypress_NEXT_UNREAD_PM_no_pm(
 
         return_value = mid_col_view.keypress(size, key)
 
+        assert not mid_col_view.controller.narrow_to_user.called
```

Comment:
Similarly this looks reasonable, but unrelated to that first commit (migration).